frontend/src/concepts/roleBinding/RoleBindingPermissionsTableRow.tsx

```diff
@@ -67,8 +71,8 @@ const ProjectSharingTableRow: React.FC<ProjectSharingTableRowProps> = ({
           )}
         </Td>
         <Td dataLabel="Permission">
-          {isEditing ? (
-            <ProjectSharingPermissionSelection
+          {isEditing && !isModelRegistry ? (
```

Comment:
On second thought, do you even really need the `hasPermissionSelection` logic? Maybe you simply only show the selection dropdown if there's more than one option, since MR will only have the single "Default" option?

```ts
isEditing && permissionOptions.length > 1
```

That would keep you from having to lift out `hasPermissionSelection` through all those component layers like I described above.

Also, it looks like you're not passing down `permissionOptions` into `RoleBindingPermissionsPermissionSelection` here.

Reply:
with the tooltip now also on the defaultGroup... might need an `isDefaultGroup` prop of some sort